src/examples/go_monorepo_setup.yml

```
@@ -0,0 +1,40 @@
+description: >
+  GO monorepo example.
+  Note: circleci-config typically consists of 2 files in this case (see also go_monorepo_workflows.yml)
+
+# Provide a use-case based example for using this orb.
+# Everything in the `usage` section will be displayed in the orb registry.
+# Comments are not retained.
+
+usage:
+
+  #############################################################################
+  # This is what you typically want to have in your .circleci/config.yml
+  #############################################################################
+
+  version: 2.1
+
+  orbs:
+    path-filtering: circleci/path-filtering@0.1.3
+
+  ###############################################################################
+  # Use of setup workflows must be enabled in circleCI project settings
+  # (Project settings > Advanced -> Dynamic config using setup workflows)
+  ###############################################################################
+
+  # this allows you to use CircleCI's dynamic configuration feature
+  setup: true
+
+  workflows:
+    generate-config:
+      jobs:
+        - path-filtering/filter:
+            base-revision: master
+            config-path: .circleci/go-monorepo-workflows.yml
+            # 3-column, whitespace-delimited mapping. One mapping per line:
+            # <regex path-to-test> <parameter-to-set> <value-of-pipeline-parameter>
+            # FIXME: replace app paths and names with your actual names/paths
+            mapping: |
+              worker/.*|.circleci/.* worker true
+              api/.*|.circleci/.* api true
+              common/.*|.circleci/.* common true
```

common=true should trigger everything - otherwise we can inline the "common=true" entirely
```suggestion
              worker/.* worker true
              api/.* api true
              common/.*|.circleci/.* common true
```